localstack/services/lambda_/invocation/docker_runtime_executor.py

```diff
@@ -349,6 +357,19 @@ def start(self, env_vars: dict[str, str]) -> None:
                 container_config.ports = PortMappings()
             container_config.ports.add(config.LAMBDA_INIT_DELVE_PORT, config.LAMBDA_INIT_DELVE_PORT)
 
+        if (
```

Comment:
This LocalStack-side chmod control is much more reliable than the previous heuristic 👍